### PR TITLE
fix: downgrade wrangler

### PR DESCRIPTION
**Pull request by LunrEclipse**
Downgrades wrangler to more stable version and removes a flakey test

### CODE REVIEW

**Review by zzmp — CHANGES_REQUESTED** [review]
Can you only make changes to `test.yml`? The other changes may or may not work to deflake, but will definitely create merge conflicts with #6902.

**Comment by LunrEclipse** [comment]
> Can you only make changes to `test.yml`? The other changes may or may not work to deflake, but will definitely create merge conflicts with #6902.

I could but I don't think the merge conflicts will be that bad. #6902 already uses the same downgraded version of wrangler and the changes to `default.test.ts` aren't touched in the dynamic image pr

**Comment by zzmp** [comment]
> > Can you only make changes to `test.yml`? The other changes may or may not work to deflake, but will definitely create merge conflicts with #6902.
> 
> I could but I don't think the merge conflicts will be that bad. #6902 already uses the same downgraded version of wrangler and the changes to `default.test.ts` aren't touched in the dynamic image pr

You're going to have merge conflicts with the yarn.lock.

**Comment by LunrEclipse** [comment]
> > > Can you only make changes to `test.yml`? The other changes may or may not work to deflake, but will definitely create merge conflicts with #6902.
> > 
> > 
> > I could but I don't think the merge conflicts will be that bad. #6902 already uses the same downgraded version of wrangler and the changes to `default.test.ts` aren't touched in the dynamic image pr
> 
> You're going to have merge conflicts with the yarn.lock.

**Review by zzmp — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -142,7 +142,8 @@
     "typescript": "^4.9.4",
     "webpack": "^5.88.2",
     "webpack-retry-chunk-load-plugin": "^3.1.1",
-    "wrangler": "^3.4.0",
+    "//": "downgraded-because-vercel-og-incompatible",
```
Comment (natew) [inline package.json:145]: can you test if we can have multple "//" or if we need to add some sort of "// wrangler" here to avoid conflicts

**Review by natew — APPROVED** [review]

**Pull request merged**